.eslintignore

```diff
@@ -1,2 +1,2 @@
-dist/js/app.js
 dist/js/app.min.js
+src/js
```

Comment:
Yes, because all of our source js is built into app.js - so if app.js is being linted, then everything is being linted.

It's better this way around, because there were linting issues cropping up for the individual js files in /src/js - e.g. functions that exist in global.js, being called by nightly.js, were being flagged as issues. So I could escape all of them, but it seemed much better to just lint app.js